build.gradle

```diff
@@ -47,5 +47,6 @@ allprojects {
         maven { url "https://plugins.gradle.org/m2/" }
         gradlePluginPortal()
         maven { url 'https://jitpack.io' }
+        mavenLocal()
```

Comment:
Remove this after removing snapshot dependencies.

Reply:
If we can safely remove jitpack.io this should be done, too.

Reply:
@kirillt We cannot remove `jipack.io` yet since there are still dependencies being hosted there, for instance: com.github.MikeOrtiz:TouchImageView. So let's keep it for now.